packages/app/cypress/e2e/runner/support/spec-loader.ts

```diff
@@ -2,6 +2,7 @@ export const shouldHaveTestResults = ({ passCount, failCount, pendingCount }) =>
   passCount = passCount || '--'
   failCount = failCount || '--'
 
+  cy.get('button.restart').should('be.visible') // ensure tests are finished running
```

Comment:
Did you need to add this for test retries?

Reply:
No, it's unrelated. I just noticed that if there's supposed to be more than one failure, but `failCount` is mistakenly set to less than the number of failures, it can result in a false positive because the fail count assertion passes as soon at there's _failCount_ failures, which is mid-test. This ensures it won't try to get the number failures and run the assertion until the tests are finished.